cli/main.py

```diff
@@ -130,7 +124,7 @@ def get_settings_files(self):
         settings_files = []
 
         # first take all input files from args
-        for input_file in self.args['input'] or []:
+        for input_file in self.args.get('input', []) or []:
```

Comment:
We need that for the case when 'input' argument it not in spec. Without that change we will get an exception saying the 'input' key is not present in input.